PianoFromAbove/MIDI.h

```diff
@@ -71,6 +71,8 @@ class MIDI
     static const int C8 = 108;
     static const int C4 = C8 - 4 * 12;
     static const int A0 = C8 - 7 * 12 - 3;
+	static const int minKey = 0;
+	static const int maxKey = 128;
```

Comment:
Change this variable to edit max key

Reply:
it will need to be 127 instead of 128 since the first note is 0, thus making it 129 keys, which the 129th key doesn't exist

Reply:
yes, and  set maxkey to 128 cause error and crash